File: src/main/java/com/fifo/ticketing/global/util/DateTimeUtil.java
package com.fifo.ticketing.global.util;

import java.time.LocalDateTime;
import java.time.ZoneId;
import java.util.Date;

public class DateTimeUtil {

    public static Date toDate(LocalDateTime localDateTime) {
        return Date.from(localDateTime.atZone(ZoneId.systemDefault()).toInstant());
    }

}

File: src/main/java/com/fifo/ticketing/global/util/DateTimeUtil.kt
package com.fifo.ticketing.global.util

import java.time.LocalDateTime
import java.time.ZoneId
import java.util.*

object DateTimeUtil {

    @JvmStatic
    fun toDate(localDateTime: LocalDateTime): Date {
        return Date.from(localDateTime.atZone(ZoneId.systemDefault()).toInstant())
    }
}
